consensus/processor.go

```diff
@@ -95,9 +106,12 @@ func NewConfiguredProcessor(validator SignedMessageValidator, rewarder BlockRewa
 func (p *DefaultProcessor) ProcessBlock(ctx context.Context, st state.Tree, vms vm.StorageMap, blk *types.Block, ancestors []types.TipSet) ([]*ApplicationResult, error) {
 	var emptyResults []*ApplicationResult
 
+	pbsw := pbTimer.Begin(ctx)
 	processBlkTimer := time.Now()
 	defer func() {
-		log.Infof("[TIMER] DefaultProcessor.ProcessBlock BlkCID: %s - elapsed time: %s", blk.Cid(), time.Since(processBlkTimer).Round(time.Millisecond))
+		dur := time.Since(processBlkTimer).Round(time.Millisecond)
+		log.Infof("[TIMER] DefaultProcessor.ProcessBlock BlkCID: %s - elapsed time: %s", blk.Cid(), dur)
+		pbsw.Stop(ctx)
```

Comment:
Great. Maybe we can return the duration from `Stop()` and avoid the need for a separate time measurement just for logging.